resolwe_bio/processes/import_data/ml_ready_reference.py

```diff
@@ -0,0 +1,156 @@
+"""Upload reference space for ML ready data."""
+import json
+import os
+from collections import Counter
+
+import pandas as pd
+
+from resolwe.process import FileField, JsonField, SchedulingClass, StringField
+from resolwe.process.models import Collection, DescriptorSchema, Entity
+
+from resolwe_bio.process.runtime import ProcessBio
+
+
+class ReferenceSpace(ProcessBio):
+    """
+    Define the reference space of ML-ready data sets.
+
+    Use a descriptive name and slug that uniquely define the reference space, e.g.:
+
+        - ComBat space of all TCGA v001
+        - Quantile transform to uniform distribution v042.
+    """
+
+    slug = "reference-space"
```

Comment:
AFAIK, we plan this to be data agnostic, e.g. we could simply use some other data (proteomics, etc.) Will leave as is for now.

Reply:
What got me thinking about this is the need for KB to be populated with the relevant features for the process to work. This is currently true for (most) transcriptomics datasets, but uploads for other modalities might not work out of the box.

Reply:
After consulting with @mstajdohar , I've modified the check now. If source is present in KB, it will made the appropriate checks, but if source is not in KB it will just issue a warning. In this way it can also support other types of features. If we decide to put those in KB